Src/Sarif.PatternMatcher/SearchSkimmer.cs

```diff
@@ -292,23 +292,27 @@ private void RunMatchExpression(FlexMatch binary64DecodedMatch, AnalyzeContext c
                             // The validator returned a bad value.
                             // TODO: we should log this condition
                             // and then continue processing.
+                            level = FailureLevel.Error;
                             break;
                         }
 
                         case Validation.Valid:
                         {
+                            level = FailureLevel.Error;
                             validationState = " which was determined to be valid";
                             break;
                         }
 
                         case Validation.Invalid:
                         {
+                            level = FailureLevel.Warning;
```

Comment:
>level [](start = 28, length = 5)

If you conclusively marked something as invalid, that's a warning only, it shouldn't ever break a process (like an error will).